Formula/metashell.rb

```diff
@@ -37,6 +37,7 @@ def install
       template <class T> struct add_const { using type = const T; };
       add_const<int>::type
     EOS
-    assert_match /const int/, shell_output("cat #{testpath}/test.hpp | #{bin}/metashell -H")
+    output = pipe_output("#{bin}/metashell -H", "cat #{testpath}/test.hpp, 0)
```

Comment:
Looks like you forgot a trailing `"` =)
20:24:01 ==> FAILED
20:24:01 Error: Invalid formula: /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/metashell.rb
20:24:01 metashell: /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/metashell.rb:41: syntax error, unexpected tIDENTIFIER, expecting ')'
20:24:01     assert_match "const int", output
20:24:01                        ^
20:24:01 /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/metashell.rb:43: unterminated string meets end of file
20:24:01 /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/metashell.rb:43: syntax error, unexpected end-of-input, expecting keyword_end

Reply:
@DomT4 I added the `"`. I hope you don't mind.